frontend/src/pages/Resources/components/ResourceSharingForm/resource-sharing-form.module.scss

```diff
@@ -301,9 +334,17 @@ input[type="date"]:not(:valid):before {
 .submit-btn {
   display: flex;
   justify-content: center;
+
   width: 40%;
   margin: 0 auto;
 }
+.Ask-div-main {
```

Comment:
Please format this file - each style should be separated by an empty line.

Reply:
Ok i will